README.md

```diff
@@ -81,6 +81,9 @@ from qiskit.circuit.library import EfficientSU2
 from scipy.optimize import minimize
 import numpy as np
 
+# set seed for reproducibility
+np.random.seed(42)
```

Comment:
This is really the legacy way and not the way to do things nowadays. I would do one of two things. The simplest would be to set some fixed initial point and not generate a random one. The other, if you want to stick with this sort of thing, would be to initialize an rng here with a seed
```
rng = np.random.default_rng(42)
```
and use this rng later to get the random values from.